java/vector/src/test/java/org/apache/arrow/vector/TestFixedSizeListVector.java

```diff
@@ -243,6 +244,29 @@ public void testTransferPair() {
     }
   }
 
+  @Test
+  public void testTransferEmptyVector() throws Exception {
+    // #43320
+    try (FixedSizeListVector src = new FixedSizeListVector("src", allocator, FieldType.nullable(new ArrowType.FixedSizeList(2)), null);
+         FixedSizeListVector dest = new FixedSizeListVector("dest", allocator, FieldType.nullable(new ArrowType.FixedSizeList(2)), null)) {
+      src.makeTransferPair(dest).transfer();
+
+      IntVector els = (IntVector) dest.addOrGetVector(FieldType.nullable(MinorType.INT.getType())).getVector();
+
+      dest.allocateNew();
+      dest.startNewValue(0);
+      els.setSafe(0, 1);
+      els.setSafe(1, 2);
+      dest.setValueCount(1);
```

Comment:
Why set to 1?

Reply:
This is a FSLV with 1 element, whose sole element is a list of 2 items.